test/augmentation/test_augmentation.py

```diff
@@ -1623,3 +1634,90 @@ def test_gradcheck(self, device):
         }
         assert gradcheck(RandomMotionBlur(
             kernel_size=3, angle=(10, 30), direction=(-0.5, 0.5)), (inp, params), raise_exception=True)
+
+
+class TestRandomEqualize:
+    def smoke_test(self, device, dtype):
```

Comment:
Recently found it shall be "test_smoke" there.

```suggestion
    def test_smoke(self, device, dtype):
```